src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonNomisSyncApiService.kt

```diff
@@ -0,0 +1,30 @@
+package uk.gov.justice.digital.hmpps.prisonerfromnomismigration.prisonperson
+
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.web.reactive.function.client.WebClient
+import org.springframework.web.reactive.function.client.awaitBodilessEntity
+
+class PrisonPersonNomisSyncApiService(@Qualifier("nomisSyncApiWebClient") private val webClient: WebClient) {
+  suspend fun syncPhysicalAttributes(prisonerNumber: String) =
```

Comment:
This will be needed for certain scenarios when implementing the move booking edge case fix.